### PR TITLE
Preload home page content for fast first paint

### PR DESCRIPTION
**Pull request by TimvdLippe**
An initial full load can now be generated and is later
inserted into the slot of home-page.

Note that I had to remove the image slider to make this all work. Since this was scheduled for removal anyways, that shouldn't be a problem.

We should remove `<lancie-section>` as well, as we can just do `:nth-child` and use `<h2>` directly.

### CODE REVIEW

File: src/lancie-home-page/lancie-home-page.html [hunk 1]
```diff
@@ -53,47 +47,15 @@
       }
     </style>
 
+    <slot></slot>
```
Comment (martijnjanssen) [inline src/lancie-home-page/lancie-home-page.html]: Above that `lancie-ajax`? Is this to prevent lancie-ajax loading first?

Comment (TimvdLippe) [reply src/lancie-home-page/lancie-home-page.html]: The issue with non-shadow dom supported browsers is that the elements are moved in the dom. Then `:nth-child` takes into account the `lancie-ajax`, e.g. flipping the colors.

Comment (martijnjanssen) [reply src/lancie-home-page/lancie-home-page.html]: Wouldn't a `:nth-child` with `:not(lancie-ajax)` work?

**Comment by martijnjanssen** [comment]
The styling is completely missing in the content that is loaded before the lancie-home. Font family is the most obvious one.

**Comment by TimvdLippe** [comment]
What do you mean with "Style completely missing"? I have a full screen paint with content that remains styled as is

**Comment by martijnjanssen** [comment]
image
This is what happens in both chrome and firefox, a different font and letter size and spacing.

**Comment by TimvdLippe** [comment]
Oh, I thought it was "completely missing". Probably missed several CSS styles, let's see.

File: index.html [hunk 2]
```diff
@@ -178,7 +251,7 @@
           <paper-icon-button class="menu-button" icon="menu" drawer-toggle></paper-icon-button>
 
           <a href="/">
-            <img height="32" width="200" src="images-optimized/lancie/logo_without_date.png" />
+            <img class="logo" height="32" width="200" src="images-optimized/lancie/logo_without_date.png" />
```
Comment (elarb) [inline index.html]: Original image is 2143x349. Maybe we should resize that one?

**Comment by martijnjanssen** [comment]
Is this completely finished? 😛

**Comment by TimvdLippe** [comment]
Um yes, if it works ;)

File: src/lancie-home-page/lancie-home-page.html (hunk 1 above)
Comment (TimvdLippe) [reply src/lancie-home-page/lancie-home-page.html]: No, it just counts the children sadly :(

**Comment by Fastjur** [comment]
So what is the status here?

Should I test this or are there some known issues still?

**Comment by TimvdLippe** [comment]
I am holding this off for now, we can do this change later.

Op ma 4 dec. 2017 om 14:42 schreef Jurriaan Den Toonder <
notifications@github.com>:

**Comment by Fastjur** [comment]
Alright :+1:

**Comment by TimvdLippe** [comment]
Have to think about this, but this is not the solution that is good enough. Therefore closing this PR.

**Pull request closed without merging**